config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml

```diff
@@ -1,24 +1,4 @@
 periodics:
-- name: periodic-cluster-api-build
-  interval: 1h
-  decorate: true
-  labels:
-    preset-service-account: "true"
-  extra_refs:
-  - org: kubernetes-sigs
-    repo: cluster-api
-    base_ref: master
-    path_alias: sigs.k8s.io/cluster-api
-  spec:
-    containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200720-4766100-1.18
-      command:
-      - "./scripts/ci-build.sh"
-  annotations:
-    testgrid-dashboards: sig-cluster-lifecycle-cluster-api
-    testgrid-tab-name: build
-    testgrid-alert-email: kubernetes-sig-cluster-lifecycle-cluster-api-alerts@googlegroups.com
-    testgrid-num-failures-to-alert: "2"
```

Comment:
Remove this job given that most of it it's being tested already within `periodic-cluster-api-test`